PHP update (#1249)
temporalio/documentation: docs/php/how-to-handle-a-query-in-a-workflow-in-php.md

`currentState`:

```php
#[Workflow\WorkflowInterface]
class MyWorkflow
{
    private string $currentState;

    #[Workflow\QueryMethod('current_state')]
    public function getCurrentState(): string
    {
        return $this->currentState;
    }

    #[Workflow\WorkflowMethod]
    public function run()
    {
        // Your normal Workflow code begins here, and you update the currentState
        // as the code makes progress.
        $this->currentState = 'waiting timer';
        try{
            yield Workflow::timer(DateInterval::createFromDateString('1 hour'));
        } catch (\Throwable $e) {
            $this->currentState = 'timer failed';
            throw $e;
        }

        $myActivity = Workflow::newActivityStub(
            MyActivityInterface::class,
            ActivityOptions::new()->withScheduleToStartTimeout(60)
        );

        $this->currentState = 'waiting activity';
        try{
            yield $myActivity->doSomething('some input');
        } catch (\Throwable $e) {
            $this->currentState = 'activity failed';
            throw $e;
        }

        $this->currentState = 'done';

        return null;
    }
}
```

> You can also issue a query from code using the `QueryWorkflow()` API on a Temporal client object.

Reviewer: ```suggestion
You can also issue a Query from code using the `QueryWorkflow()` API on a Temporal Client object.
```
```suggestion
You can also issue a query from code using the `QueryWorkflow()` API on a Temporal client object.
```

> Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to running Workflows as well):

Reviewer: ```suggestion
Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to both running and closed Workflows):
```
```suggestion
Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to running Workflows as well):
```
